src/main/include/Behaviours/ClimberBehaviour.h

```diff
@@ -0,0 +1,21 @@
+// Copyright (c) 2023-2024 CurtinFRC
+// Open Source Software, you can modify it according to the terms
+// of the MIT License at the root of this project
+
+#pragma once
+
+#include <frc/XboxController.h>
+
+#include "Climber.h"
+#include "Wombat.h"
+
+class ClimberManualControl : public behaviour::Behaviour {
+ public:
+  explicit ClimberManualControl(Climber* climber, frc::XboxController* codriver);
+  void OnTick(units::second_t dt) override;
+
+ private:
+  Climber* _climber;
+  frc::XboxController* _codriver;
+  bool _rawControl = false;
```

Comment:
You already have this state from the _climber->GetState don't replicate it with booleans

Reply:
what does it mean replicating with booleans

Reply:
You are assigning the value of checking _climber->GetState with your desired state to _rawControl instead of directly checking it. This duplicates the code.